dist/gcc-compatible/internal/Hacl_SHA3.h

```diff
@@ -0,0 +1,52 @@
+/* MIT License
```

Comment:
you might want to wait for my other PR to land first, as it will make this file disappear, I think

Reply:
Oh year, I was wondering why this file was being generated. Happy to have it go away.

Reply:
yeah the file is still there because EverCrypt.Hash repeatedly calls `absorb_inner` in the update_multi loop, and this is a call across translation units, so it generates an internal header

(I forgot about that caller)

Reply:
I'll merge this PR, but if the internal header is bad, and you need loadState to be marked as `static` for perf reasons, open up an issue and I'll fix it